src/components/rating/Rating.jsx

```diff
@@ -21,25 +25,51 @@ class Rating extends Component {
     if (ratedStarIndex !== rate) {
       onChange(ratedStarIndex);
     }
-  }
+  };
+
+  onMouseEnter = () => {
+    const {active} = this.props;
+
+    if (!active) {
+      return;
+    }
+
+    this.setState({showActiveText: true});
+  };
+
+  onMouseLeave = () => {
+    this.setState({showActiveText: false});
+  };
 
   render() {
-    const {metricSize = 5, rate = 0, small, active, counter, className} = this.props;
+    const {metricSize = 5, rate = 0, small, active, className, counterText, activeText} = this.props;
+    const {showActiveText} = this.state;
     const ratingClass = classnames('sg-rate-box', {
       'sg-rate-box--small': small,
       'sg-rate-box--active': active
     }, className);
+
     const starsProps = new Array(metricSize).fill(true).map((dump, index) => ({
+      index,
       key: index,
       checked: index < rate,
       size: small ? ICO_SIZE.SMALL : ICO_SIZE.NORMAL,
-      onClick: () => this.onClick(index)
+      onClick: this.onClick,
+      onMouseEnter: this.onMouseEnter,
+      onMouseLeave: this.onMouseLeave
     }));
 
+    const rateString = rate.toLocaleString(undefined, {minimumFractionDigits: 1, maximumFractionDigits: 1});
```

Comment:
`toLocaleString`, `minimumFractionDigits`, `maximumFractionDigits` - fancy! What's the support? Is it polyfilled by polyfillio? Can we have null instead of undefined?

Reply:
API working everywhere.
It give us '2,0' instead of '2.0' on PL browser.